h5p-bildetema-words-theme-image/src/components/ThemeImageContainer/ThemeImageContainer.module.scss

```diff
@@ -1,7 +1,7 @@
 @use "../../../../common/abstracts" as *;
 
 .wrapper {
-  display: flex;
+  display: -webkit-box;
```

Comment:
??

```suggestion
  display: flex;
```

Reply:
with display flex, the boxes were never side by side